artiq/frontend/artiq_dashboard.py

```diff
@@ -38,9 +35,8 @@ def get_argparser():
         "--port-broadcast", default=1067, type=int,
         help="TCP port to connect to for broadcasts")
     parser.add_argument(
-        "--db-file", default=default_db_file,
-        help="database file for local GUI settings "
-             "(default: %(default)s)")
+        "--db-file", default=None,
+        help="database file for local GUI settings")
```

Comment:
I don't like losing the path to the file. Maybe add "by default in ... and dependent on master hostname".

Reply:
OK